website/docs/execution-node/fee-recipient.md

```diff
@@ -59,21 +59,21 @@ A fee recipient address can be configured on your client instance by using one o
     <td>
     Sets a default ETH address for all validator public keys. <br /> <br /> 
     <strong>Example</strong>: <code>--suggested-fee-recipient=0x0123456722E6b0000012BFEBf6177F1D2e9758D9</code> <br /> <br /> 
-    <strong>Note</strong>: This setting overrides the two config options below. If you set this, the config options will be ignored.
+    <strong>Note</strong>: This setting is overwritten by the flags below and will act as a fallback catch all.
```

Comment:
```suggestion
    <strong>Note</strong>: This setting is overwritten by the flags below. If you don't set any other flags, this will apply to all validator public keys.
```

Thoughts on this?